test/support/helpers/platform.js

```diff
@@ -27,11 +26,11 @@ module.exports = {
 //     })
 //
 function onMacOSAtLeast (minRelease /*: MacOSReleaseInfo */, spec /*: Function */) {
-  const describe = macOSRelease.isAtLeast(minRelease)
-    ? mocha.describe
-    : mocha.describe.skip
+  const describeOverwritten = macOSRelease.isAtLeast(minRelease)
```

Comment:
`describeOrSkip`?

Reply:
Why not, at least it assumes the bad idea of overwriting a function.